tests/wdio.conf.ts

```diff
@@ -0,0 +1,260 @@
+import AllureReporter from '@wdio/allure-reporter';
+
+import { getLogs, initLogger, logInfo } from './helpers/browserLogger';
+
+const allure = require('allure-commandline');
+
+// This is deprecated without alternative (https://github.com/nodejs/node/issues/32483)
+// we need it to be able to reuse jitsi-meet code in tests
+require.extensions['.web.ts'] = require.extensions['.ts'];
+
+const chromeArgs = [
+    '--allow-insecure-localhost',
+    '--use-fake-ui-for-media-stream',
+    '--use-fake-device-for-media-stream',
+    '--disable-plugins',
+    '--mute-audio',
+    '--disable-infobars',
+    '--autoplay-policy=no-user-gesture-required',
+    '--auto-select-desktop-capture-source=Your Entire screen',
+    '--no-sandbox',
+    '--disable-dev-shm-usage',
+    '--disable-setuid-sandbox',
+    '--use-file-for-fake-audio-capture=tests/resources/fakeAudioStream.wav'
+];
+
+if (process.env.RESOLVER_RULES) {
+    chromeArgs.push(`--host-resolver-rules=${process.env.RESOLVER_RULES}`);
+}
+if (process.env.ALLOW_INSECURE_CERTS === 'true') {
+    chromeArgs.push('--ignore-certificate-errors');
+}
+if (process.env.HEADLESS === 'true') {
+    chromeArgs.push('--headless');
+    chromeArgs.push('--window-size=1400,600');
```

Comment:
Why the weird resolution? Why not 1080p?

Reply:
🤷 I will change it, this is what torture runs today.